spec/features/consumer/shopping/checkout_spec.rb

```diff
@@ -160,8 +160,7 @@ def fill_out_form
         let!(:stripe_pm) do
           create(:stripe_payment_method,
             distributors: [distributor],
-            name: "Stripe",
-            preferred_enterprise_id: distributor.id)
+            name: "Stripe")
```

Comment:
No need for this name. It is the default.

Reply:
Correct, the name actually doesn't matter for this spec at all. I removed it. Have a look at my new commit.